<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSDConfigBase</class>
 <widget class="QWidget" name="OSDConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkMessage">
     <property name="text">
      <string>Enable &amp;messages notification</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkStatus">
     <property name="text">
      <string>Enable &amp;status notification</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkStatusOnline">
     <property name="text">
      <string>Online</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="chkStatusAway">
     <property name="text">
      <string>Away</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="chkStatusNA">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="chkStatusDND">
     <property name="text">
      <string>Do not Disturb</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkStatusOccupied">
     <property name="text">
      <string>Occupied</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chkStatusFFC">
     <property name="text">
      <string>Free for chat</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="chkStatusOffline">
     <property name="text">
      <string>Offline</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkTyping">
     <property name="text">
      <string>Enable &amp;typing notification</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="chkMessageContent">
     <property name="text">
      <string>Show message &amp;content</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="lblLines">
     <property name="text">
      <string>Show lines of content:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="edtLines">
     <property name="specialValueText">
      <string>All</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="chkCapsLockFlash">
     <property name="text">
      <string>Enable keyboard LED-notification in N/A state</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="Spacer5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="2">
    <spacer name="Spacer29">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
